### ISSUE TITLE
Graph line blends in very well with axis lines? (solution: darker fill area?)

### ISSUE THREAD

**Issue by MoneroChan** [M5M400/supportxmr-gui#13]
Hi, 
I notice the graph line blends in really well with the horizontal axis lines, 
It may be hard to see for some people with a steady hashrate coupled with a High res monitor. 

Proposed solution:
-  https://imgur.com/a/iealr8P

(changing the fill colour under the line)

I'm not too fussed, just a thought

cheers,

**Comment by Thunderosa** [comment]
Yep, good point. I'll work some contrast in for the next update. Thanks!
